egs/librispeech/asr/simple_v1/README.md

```diff
@@ -0,0 +1,132 @@
+
+
+## k2_librispeech
+An example of how to build G and L FST for K2.
+
+Most scripts of this example are copied from Kaldi.
+
+### Run scripts
+```bash
+$ ./run.sh
+
+$ ls data/lang_nosp
+
+G.fsa.txt
+L.fst.txt
+L_disambig.fst.txt
+oov.int
+oov.txt
+phones
+phones.txt
+words.txt
+```
+
+### Load L, G into K2
+```python
+import k2, _k2
+
+
+with open('data/lang_nosp/L.fst.txt') as f:
+    s = f.read()
+
+Lfst = k2.Fsa.from_openfst(s, acceptor=False)
+
+with open('data/lang_nosp/G.fsa.txt') as f:
+    s = f.read()
+
+Gfsa = k2.Fsa.from_openfst(s, acceptor=True)
+```
+
+### An example of G building
+The `toy.arpa` file:
+```plain
+\data\
+ngram 1=5
+ngram 2=6
+ngram 3=1
+
+\1-grams:
+-2.348754 </s>
+-99 <s> -1.070027
+-4.214113 A -0.5964623
+-4.255245 B -0.3214741
+-4.20255  C -0.2937318
+
+\2-grams:
+-4.284099 <s> A -0.1969815
+-1.100091 A </s>
+-2.839235 A B -0.1747991
+-2.838903 A C -0.5100551
+-1.104238 B </s>
+-1.251644 C </s>
+
+\3-grams:
+-0.1605104  A C B
+
+\end\
+```
+
+Build G fst:
+```bash
+$ local/arpa2fst.py toy.arpa
+
+0 1 </s> -5.408205947510138
+2 0 <eps> -1.070027
+0 3 A -9.703353773992418
+3 0 <eps> -0.5964623
+0 4 B -9.79806370403745
+4 0 <eps> -0.3214741
+0 5 C -9.676728982562127
+5 0 <eps> -0.2937318
+2 6 A -9.864502494310699
+6 3 <eps> -0.1969815
+3 1 </s> -2.5330531375369127
+3 7 B -6.53758018650695
+7 4 <eps> -0.1747991
+3 8 C -6.536815728256077
+8 5 <eps> -0.5100551
+4 1 </s> -2.5426019579175594
+5 1 </s> -2.8820168161354394
+8 9 B -0.36958885431051147
+1
+```
+
+Draw it by Graphviz:
```

Comment:
Wondering how to obtain the following output since it is not described
in this document.

Reply:
@jimbozhang  might be able to comment on that.

Reply:
I would suggest copying it to
https://dreampuf.github.io/GraphvizOnline

and create a short URL via https://git.io/

A visualization of this sample G.fst is available at 
https://git.io/JkqnK

Reply:
Deleted as we don't really need that